rootfs-builder/rootfs.sh

```diff
@@ -368,6 +368,14 @@ popd  >> /dev/null
 
 [ -n "${KERNEL_MODULES_DIR}" ] && copy_kernel_modules ${KERNEL_MODULES_DIR} ${ROOTFS_DIR}
 
+chrony_conf_file="${ROOTFS_DIR}/etc/chrony.conf"
+if [ ${distro} == ubuntu ] || [ ${distro} == debian ] ; then
+	chrony_conf_file="${ROOTFS_DIR}/etc/chrony/chrony.conf"
+fi 
+
+info "Configure chrony file ${chrony_conf_file}"
+echo "refclock PHC /dev/ptp0 poll 3 dpoll -2 offset 0" >> ${ROOTFS_DIR}/etc/chrony.conf
```

Comment:
This looks wrong for the Ubuntu/Debian case. Shouldn't it be something like:

```sh
mkdir -p $(dirname "$chrony_conf_file")
echo "refclock PHC /dev/ptp0 poll 3 dpoll -2 offset 0" >> "chrony_conf_file"
```

Reply:
@jodh-intel That directory is created by the package install for ubuntu/debian.  They install the chrony config file in `/etc/chrony/` instead of `/etc`.
I think we should skip creating that directory, so that we are informed of an error in case the directory is not created by distro install. wdyt @jodh-intel ?

I also realised that the service is called `chrony.service` in debian/ubuntu whereas it is called `chronyd.service` in all other distros. Will see how to handle this. Suggestions are welcome.
I am planning to add the service here:
https://github.com/kata-containers/agent/blob/master/kata-containers.target#L3

Reply:
@jodh-intel I have fixed this now